pkgs/development/python-modules/pytest-subprocess/default.nix

```diff
@@ -0,0 +1,45 @@
+{ lib
+, buildPythonPackage
+, pythonOlder
+, fetchFromGitHub
+, pytest
+, pytestCheckHook
+, docutils
+, pygments
+}:
+
+buildPythonPackage rec {
+  pname = "pytest-subprocess";
+  version = "1.0.1";
+
+  disabled = pythonOlder "3.4";
+
+  src = fetchFromGitHub {
+    owner = "aklajnert";
+    repo = "pytest-subprocess";
+    rev = version;
+    sha256 = "16ghwyv1vy45dd9cysjvcvvpm45958x071id2qrvgaziy2j6yx3j";
+  };
+
+  buildInputs = [
+    pytest
+  ];
```

Comment:
Why is this required? Maybe we can just patch it out? If it is required to build the package it needs to be nativeBuildInputs.

Reply:
It's in `install_requires`.
But we support different versions of `pytest`.